editor/components/block-list/index.js

```diff
@@ -211,20 +211,16 @@ class BlockList extends Component {
 		return (
 			<div>
 				{ !! blocks.length && <BlockListSiblingInserter /> }
-				{ flatMap( blocks, ( uid ) => [
+				{ flatMap( blocks, ( uid ) => (
```

Comment:
This can be `map` now instead of `flatMap` if we're just returning a single value from the mapper.